### ISSUE TITLE
Any modifications required for the Google Home Mini?

### ISSUE THREAD

**Issue by donn-smith** [bjoernkarmann/project_alias#16]
I and several coworkers are interested in building these for our personal use. Has anyone tried it with a Google Home Mini as that seems to be the smart speaker of choice in our group? Are the microphones of the mini spaced similarly to the Google Home?

**Comment by dragon788** [comment]
With the dome shape on the top of the Google Home Mini the 3D printed part will likely not properly interfere with the microphones to prevent it from hearing things.

**Comment by dragon788** [comment]
See https://www.ifixit.com/Teardown/Google+Home+Mini+Teardown/102264 for actual locations.

**Comment by donn-smith** [comment]
Then we'll have to develop our own holder. Thanks for the pointers to the teardown. It did confirm where I thought the microphones are.

**Comment by espoelstra** [comment]
See also this comparison shot of the mic board of the mini on top of the regular Google Home.

https://drive.google.com/file/d/1y5IdLOxlhdWW_A8DrJrClvi78OX2GwJ6Fw/view?usp=sharing

**Comment by espoelstra** [comment]
There are some tools like Fusion 360 that support taking photos and generating a 3D model from the photos, perhaps someone can do this for the Mini and share the models in this repo or via Thingiverse or something similar.

**Comment by filirnd** [comment]
Hi, I'm interessed for the google mini version of Alias. Have you make an opensource 3d model for this ?

**Comment by donn-smith** [comment]
I have not and no one has contacted me about doing so. I do not have access to a 3D printer at this time, so I am not pursuing this design right now.

**Comment by f3rm4rf3r** [comment]
Any update on this? I am looking for a google home compatible hat. Thanks!

**Comment by dragon788** [comment]
The new Nest Mini has a slightly different shape and may have a different mic layout.

**Comment by CestMoiRoma** [comment]
The nest mini have 3 mics but alias use only 2 speaker, i got a nest mini i can test to make the 3D file but i need to know if i need to cover the 3 mics with speakers or if 2 speaker will do the job ?

**Comment by OFFCANNY** [comment]
> The nest mini have 3 mics but alias use only 2 speaker, i got a nest mini i can test to make the 3D file but i need to know if i need to cover the 3 mics with speakers or if 2 speaker will do the job ?

Hi, I'm also trying to make a tweaked version for Nest Mini. From my research, the Nest Mini also only has 2 speakers. ( https://youtu.be/UdFTvebhk_0?t=178 )

From the info above from other users, I overlayed the Nest Mini on top of the Home, and you can see that the Pi's speakers need to be positioned more inwards. https://postimg.cc/gallery/Z3bpwWY 
![Nest_Mini_Atlas 1](https://user-images.githubusercontent.com/80494002/110875781-45aac900-82a4-11eb-88fe-2aa4e148f81b.png)
![Nest_Mini_Atlas 2](https://user-images.githubusercontent.com/80494002/110875803-4f343100-82a4-11eb-8a1a-265caf8777e1.png)

**Comment by OFFCANNY** [comment]
> > The nest mini have 3 mics but alias use only 2 speaker, i got a nest mini i can test to make the 3D file but i need to know if i need to cover the 3 mics with speakers or if 2 speaker will do the job ?
> 
> Hi, I'm also trying to make a tweaked version for Nest Mini. From my research, the Nest Mini also only has 2 speakers. ( https://youtu.be/UdFTvebhk_0?t=178 )
> 
> From the info above from other users, I overlayed the Nest Mini on top of the Home, and you can see that the Pi's speakers need to be positioned more inwards. https://postimg.cc/gallery/Z3bpwWY
> ![Nest_Mini_Atlas 1](https://user-images.githubusercontent.com/80494002/110875781-45aac900-82a4-11eb-88fe-2aa4e148f81b.png)
> ![Nest_Mini_Atlas 2](https://user-images.githubusercontent.com/80494002/110875803-4f343100-82a4-11eb-8a1a-265caf8777e1.png)

Correction, the Home Mini has 2 Mics, the NEST MINI does indeed have 3!

**Comment by OFFCANNY** [comment]
> The nest mini have 3 mics but alias use only 2 speaker, i got a nest mini i can test to make the 3D file but i need to know if i need to cover the 3 mics with speakers or if 2 speaker will do the job ?

**Hey guys, Im very interested in designing a version that will work on the Nest Mini. Lets do this.** 

@donn-smith Did you create one?

@Steepy12 Yes it does have 3 mics, we will need to add another speaker output. Also it doesn't look like it will be that different, the speakers only need to be a few CM inwards.  Will def need a new design. 
![Nest_Mini_Atlas 1](https://user-images.githubusercontent.com/80494002/110877044-ba7f0280-82a6-11eb-8258-c549eadaf1b7.png)
![Nest_Mini_Atlas 2](https://user-images.githubusercontent.com/80494002/110877057-c1a61080-82a6-11eb-960c-4d92aee034d6.png)
![Nest_Mini_Atlas 3](https://user-images.githubusercontent.com/80494002/110877027-b357f480-82a6-11eb-84c6-f0fbe5433eea.png)

**Comment by CestMoiRoma** [comment]
Hmm I see, adding the 3rd speaker wouldn't be complicated because the 3d model would be difficult.  Especially if there are a few internal cm to know, it would be necessary to be able to dismantle it to measure the distance, or to see to completely redo a new box with the mini nest which would simplify the task of creating the module and of everything integrated in a single box, I  will look to make a new box already tested if Alias ​​works and especially to create the 3D model

**Comment by OFFCANNY** [comment]
I don't think we need to measure exactly as long as the speaker is somewhat above the mic there should be enough white noise to make it work

**Comment by OFFCANNY** [comment]
> Hmm I see, adding the 3rd speaker wouldn't be complicated because the 3d model would be difficult. Especially if there are a few internal cm to know, it would be necessary to be able to dismantle it to measure the distance, or to see to completely redo a new box with the mini nest which would simplify the task of creating the module and of everything integrated in a single box, I will look to make a new box already tested if Alias ​​works and especially to create the 3D model

I don't think we need to measure exactly as long as the speaker is somewhat above the mic there should be enough white noise to make it work

**Comment by CestMoiRoma** [comment]
yes it could work, but second problem, how do we so that the sound of the speaker is not blocked by the "shell" which will hold the mini speaker

**Comment by CestMoiRoma** [comment]
if @donn-smith made is own file it could works, the nest and the home mini have the same emplacement for the mic on the left and right.

**Comment by donn-smith** [comment]
Sorry, I did not. I don't own a 3D printer, so I have not attempted this as of yet.
